src/nat/cli/commands/workflow/workflow_commands.py

```
@@ -122,235 +93,219 @@ def find_package_root(package_name: str) -> Path | None:
-        # Get the repository root
         try:
             repo_root = get_repo_root()
         except PackageError:
             repo_root = None
 
-        # Get the absolute path for the output directory
-        if not os.path.isabs(workflow_dir):
-            workflow_dir = os.path.abspath(workflow_dir)
+        workflow_dir = Path(workflow_dir).resolve()
+        if not workflow_dir.exists():
+            raise click.BadParameter(f"Invalid workflow directory: {workflow_dir}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)